cirq/sim/sparse_simulator_test.py

```diff
@@ -900,3 +899,44 @@ def test_random_seed_mixture_deterministic():
                   [[1], [0], [0], [0], [1], [0], [0], [1], [1], [1], [1], [1],
                    [0], [1], [0], [0], [0], [0], [0], [1], [0], [1], [1], [0],
                    [1], [1], [1], [1], [1], [0]])
+
+
+def test_entangled_reset_does_not_break_randomness():
+    # A previous version made the mistake of assuming that it was okay to cache
```

Comment:
Do you think this should use the triple quote doc string instead of the line comments?

Reply:
Huh. I never considered putting a doc string on a test method.

Reply:
Made it a doc string.